daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf2.proto

```diff
@@ -1393,15 +1393,14 @@ message InternedDottedName {
   repeated int32 segments_interned_str = 1;
 }
 
-// *Available in versions >= 2.dev*
 message UpgradedPackageId {
   int32 upgraded_package_id_interned_str = 1;
 }
 
 message PackageMetadata {
   int32 name_interned_str = 1;
   int32 version_interned_str = 2;
-  UpgradedPackageId upgraded_package_id = 1001; // *optional*, *Available in versions >= 2.dev*
```

Comment:
I think this should still be optional - the initial version of a package can't upgrade anything